Tests/FluentAssertions.Specs/Types/TypeAssertionSpecs.cs

```diff
@@ -136,6 +136,8 @@ public class ClassWithoutMembers { }
 
     public interface IPublicInterface { }
 
+    internal interface IInternalInterface { }
```

Comment:
🔧 I suggest to keep this test class with the test cases.

Reply:
The type has to be non-nested to replicate the issue.